content/docs/started/getting-started.md

```diff
@@ -46,6 +45,78 @@ to suit your environment (desktop or server, existing Kubernetes cluster or publ
   * To use Kubeflow on IBM Cloud Private (ICP),
 	follow the [ICP deployment guide](/docs/started/cloud/getting-started-icp/).
 
+## Configuration quick reference
+
+Below is a matrix of the platforms where you can deploy Kubeflow and the 
+corresponding manifest files that specify the default configuration for each
+platform. The matrix shows the same manifests as the installation guides.
+The matrix is therefore an alternative way of accessing the information in the
+[installation section above](#installation-guides).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Deployment platform</th>
+        <th>Manifest</th>
+        <th>Deployment guide</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><b>Existing Kubernetes cluster</b>: vanilla Kubeflow with no external 
```

Comment:
Nit: I don't mind "vanilla" at all but if a word like "basic" or "standard" would still be accurate, that may be better for people less familiar with English.

Reply:
Thanks! Done.